src/banbroadcaster/functions.py

```diff
@@ -1,34 +1,30 @@
-import os
-import time
+import logging
 import operator
+import time
 from collections import namedtuple
 from datetime import date, datetime
 from typing import List, NamedTuple
 
 import mysql.connector
 import tweepy
-import logging
+from config import (ACCESS_TOKEN, ACCESS_TOKEN_SECRET, CONSUMER_KEY,
```

Comment:
The general rule for imports is that you import modules (read classes) and not variables or functions, please import config and use config.CONSTANT_VARIABLE